package.json

```diff
@@ -19,64 +20,60 @@
+    "@skyux/layout": "^4.0.0-rc.1",
+    "angular-tree-component": "^8.5.6"
   },
   "dependencies": {},
   "devDependencies": {
-    "@angular/animations": "7.2.16",
-    "@angular/common": "7.2.16",
-    "@angular/compiler": "7.2.16",
-    "@angular/compiler-cli": "7.2.16",
-    "@angular/core": "7.2.16",
-    "@angular/forms": "7.2.16",
-    "@angular/http": "7.2.16",
-    "@angular/platform-browser": "7.2.16",
-    "@angular/platform-browser-dynamic": "7.2.16",
-    "@angular/router": "7.2.16",
+    "@angular/animations": "9.1.2",
+    "@angular/common": "9.1.2",
+    "@angular/compiler": "9.1.2",
+    "@angular/compiler-cli": "9.1.2",
+    "@angular/core": "9.1.2",
+    "@angular/forms": "9.1.2",
+    "@angular/platform-browser": "9.1.2",
+    "@angular/platform-browser-dynamic": "9.1.2",
+    "@angular/router": "9.1.2",
     "@blackbaud/auth-client": "2.24.0",
-    "@pact-foundation/pact": "9.9.5",
-    "@pact-foundation/pact-web": "7.4.0",
-    "@skyux-sdk/builder": "3.16.2",
+    "@blackbaud/skyux-lib-clipboard": "2.0.0-rc.0",
+    "@blackbaud/skyux-lib-code-block": "2.0.0-rc.1",
+    "@blackbaud/skyux-lib-media": "2.0.0-rc.0",
+    "@blackbaud/skyux-lib-stache": "4.0.0-rc.1",
+    "@skyux-sdk/builder": "4.0.0-rc.5",
     "@skyux-sdk/builder-plugin-skyux": "1.3.1",
-    "@skyux-sdk/e2e": "3.1.2",
-    "@skyux-sdk/pact": "3.3.1",
-    "@skyux-sdk/testing": "3.2.0",
-    "@skyux/animations": "3.0.0",
-    "@skyux/assets": "3.1.0",
-    "@skyux/avatar": "3.1.0",
-    "@skyux/config": "3.8.1",
-    "@skyux/core": "3.14.2",
-    "@skyux/errors": "3.2.0",
-    "@skyux/forms": "3.6.3",
-    "@skyux/http": "3.9.0",
-    "@skyux/i18n": "3.10.0",
-    "@skyux/indicators": "3.1.3",
-    "@skyux/inline-form": "3.1.0",
-    "@skyux/layout": "3.9.0",
-    "@skyux/lists": "3.8.5",
-    "@skyux/modals": "3.2.0",
-    "@skyux/navbar": "3.0.1",
-    "@skyux/omnibar-interop": "3.1.0",
-    "@skyux/popovers": "3.5.2",
-    "@skyux/router": "3.1.1",
-    "@skyux/theme": "3.12.0",
-    "@types/core-js": "2.5.3",
-    "@types/lodash": "^4.14.136",
+    "@skyux-sdk/e2e": "4.0.0-rc.0",
+    "@skyux-sdk/testing": "4.0.0-rc.1",
+    "@skyux/animations": "4.0.0-rc.0",
+    "@skyux/assets": "4.0.0-rc.1",
+    "@skyux/config": "4.0.0-rc.2",
+    "@skyux/core": "4.0.0-rc.4",
+    "@skyux/docs-tools": "4.0.0-rc.1",
+    "@skyux/forms": "4.0.0-rc.0",
+    "@skyux/http": "4.0.0-rc.3",
+    "@skyux/i18n": "4.0.0-rc.3",
+    "@skyux/indicators": "4.0.0-rc.3",
+    "@skyux/inline-form": "4.0.0-rc.0",
+    "@skyux/layout": "4.0.0-rc.1",
+    "@skyux/lists": "4.0.0-rc.1",
+    "@skyux/lookup": "4.0.0-rc.0",
+    "@skyux/modals": "4.0.0-rc.2",
+    "@skyux/omnibar-interop": "4.0.0-rc.0",
+    "@skyux/popovers": "4.0.0-rc.1",
+    "@skyux/router": "4.0.0-rc.0",
+    "@skyux/tabs": "4.0.0-rc.0",
+    "@skyux/theme": "4.0.0-rc.2",
     "angular-tree-component": "8.5.6",
-    "codelyzer": "4.5.0",
+    "codelyzer": "5.2.2",
     "core-js": "2.6.11",
```

Comment:
`core-js` and `@types/core-js` can be removed, since they're handled by Builder now.

Reply:
Done.